src/ci/bin/test.inc.sh

```diff
@@ -79,6 +78,8 @@ cromwell::private::create_build_variables() {
       CROMWELL_BUILD_IS_VIRTUAL_ENV=false
     fi
 
+    CROMWELL_RUN_TESTS=true
```

Comment:
```suggestion
    CROMWELL_BUILD_RUN_TESTS=true
```

All variables for the build are prefixed as `CROMWELL_BUILD_` to distinguish that the variable is affecting the **cromwell repository build**, versus just building the **cromwell server executable**.